### PR TITLE
Refactoring modules (BlockStorage, Hash)

### PR DESCRIPTION
**Pull request by MichaelKim20**
I changed the export form of modules.
I changed the code style.

### CODE REVIEW

**Comment by AndrejMitrovic** [comment]
Refactoring means changing the code without changing the functionality. But the PR introduces a database schema change.

File: tests/Storage.test.ts [hunk 1]
```diff
@@ -309,18 +309,19 @@ function putAllBlockData ()
     var idx = 0;
     var doPut = () =>
     {
+        if (idx >= sample_data.length)
+        {
+            getBlockData(1);
+                return;
+        }
+
```
Comment (Geod24) [inline tests/Storage.test.ts]: This comment is on the first commit. You should add a commit description.
The commit description needs to explain why this change is necessary.
The diff only shows some code being moved. If there was a problem, it should be described.

Comment (MichaelKim20) [reply tests/Storage.test.ts]: I will add this to the next PR. Thanks.

**Comment by MichaelKim20** [comment]
> Refactoring means changing the code without changing the functionality. But the PR introduces a database schema change.

I will separate the commit into three PRs.
So I will solve this problem.

File: tests/Storage.test.ts [hunk 2]
```diff
@@ -329,8 +329,9 @@ function putAllBlockData ()
 /**
  * Gets one block data
  */
-function getBlockData (height : number)
+function getBlockData (height: any)
```
Comment (Geod24) [inline tests/Storage.test.ts:332]: Why `any` ?

Comment (MichaelKim20) [reply tests/Storage.test.ts:332]: I want to process this as a string.
The data type of 'height' is  `ulong` and it is difficult to process in TypeScript. 
Therefore, it is for processing as a string.
It should use `bigint` to be exact, but it would be more efficient not to convert what can be treated as a string.

File: tests/Storage.test.ts (hunk 1 above)
Comment (MichaelKim20) [reply tests/Storage.test.ts]: https://github.com/bpfkorea/stoa/pull/12
I created a new PR for this.

File: tests/Storage.test.ts (hunk 2 above)
Comment (Geod24) [reply tests/Storage.test.ts:332]: I see. I am not a fan of this approach, because numbers should be stored as numbers, not string, but if it's easier in TypeScript, let's go with this.
